### PR TITLE
FEATURE: Authorization

### PR DESCRIPTION
**Pull request by osmanaliyardim**
### [Task 3](https://learn.epam.com/myLearning/program?groupGuid=68c6d8bc-9684-4ee5-a549-048b3ef481d8) - Authorization

* Add authorization
* Add users controller

* Added users controller to show all registered users.
* Added authorization rule for only administrative people to reach that specific page.

* Tested on localhost.

### CODE REVIEW

File: WebNetSample/Controllers/UsersController.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+
+namespace WebNetMVC.Controllers
+{
+    [Authorize(Roles = "Administrator")]
```
Comment (dotnet-fizzyy) [inline WebNetSample/Controllers/UsersController.cs]: Let's put into a constant

**Review by dotnet-fizzyy — COMMENTED** [review]
Could you, please, attach screenshot with confirmation where response is `403` if user not an _Administrator_ role?

**Comment by osmanaliyardim** [comment]
> Could you, please, attach screenshot with confirmation where response is `403` if user not an _Administrator_ role?

Here you are;

**Review by dotnet-fizzyy — APPROVED** [review]
Well done!

**Pull request merged**